advanced_alchemy/repository/_util.py

```diff
@@ -179,15 +275,26 @@ def _apply_filters(
         apply_pagination: bool = True,
         statement: StatementTypeT,
     ) -> StatementTypeT:
-        """Apply filters to a select statement.
+        """Apply filters to a SQL statement.
 
         Args:
-            *filters: filter types to apply to the query
-            apply_pagination: applies pagination filters if true
-            statement: select statement to apply filters
+            *filters: Filter conditions to apply. Can be:
+                - :class:`~advanced_alchemy.filters.StatementFilter` instances
```

Comment:
I don't think the doc string here should have the types. IIRC, there is a sphinx extension that auto populates this in the generated docs